perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -103,7 +103,7 @@ def _RunPing(sending_vm, receiving_vm, receiving_ip, ip_type):
   Returns:
     A list of samples, with one sample for each metric.
   """
-  if not sending_vm.IsReachable(receiving_vm):
+  if not sending_vm.IsReachable(receiving_vm) and ip_type == 'internal':
```

Comment:
Since IsReachable() is only valid on ip_type internal, consider inverting order of clauses?

Reply:
changed